Description: Mark the maven-bundle-plugin as a extension to handle the 'bundle' packaging
 (this parameter is defined in a parent pom not available in Debian)
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed
--- a/pom.xml
+++ b/pom.xml
@@ -76,6 +76,7 @@
       <plugin>
         <groupId>org.apache.felix</groupId>
         <artifactId>maven-bundle-plugin</artifactId>
+        <extensions>true</extensions>
         <version>2.0.1</version>
         <configuration>
           <instructions>
